src/lib/y2storage/callbacks/activate_callbacks.rb

```diff
@@ -21,18 +21,27 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "yast"
 require "storage"
+require "y2storage/dialogs/callbacks/activate_luks"
 
 module Y2Storage
-  # class to implement callbacks used during activate in ruby
+  # class to implement callbacks used during activate
   class ActivateCallbacks < Storage::ActivateCallbacks
+    include Yast::Logger
```

Comment:
@aschnell Yes, and the only definitive point on that conversation was to follow the Ruby/YaST convention of having 1:1 relationship between namespaces and directories. Something that we finally didn't do:

```
<ancorgs>  I fine with creating a Y2Storage::Callbacks namespace (and, thus, subdirectory) then.
           As you wish
<ancorgs>  as said, without the full picture I don't have a strong opinion, whatever you decide
<ancorgs>  just make sure every namespace has its corresponding directory so the structures of the
           files tree matches the structure of the namespaces, as is standard in the Ruby world
<ancorgs>  CamelCase for the namespaces and snake_case for the equivalent directory. That's all
<arvin>    so e.g. src/lib/y2storage/callbacks/activate.rb
<ancorgs>  yes
<ancorgs>  which will define the Y2Storage::Callbacks::Activate class
```